src/views/ABIEncoder.js

```diff
@@ -0,0 +1,142 @@
+            if (res){
+                this.setState({ result: res })
+            }else{
+                this.setState({ result: "" })
+            }
+        }
+        catch(err){
+            this.setState({ result: "[ERROR] PLEASE USE THE CORRECT FORMAT OF INPUTS!" })
+        }
+    }
+
+    render() {
+        const { argumentTypes, argumentValues, result } = this.state;
+        return (
+            <Column>
+                <Card>
+                    <CardHeader>
+                        <h3 className="title">ABI Encoder</h3>
+                        <p style={{color:"#6c757d"}}>
+                            The tool was designed to make easy encoding of Klaytn solidity ABI data.
+                        </p>
+                    </CardHeader>
+                    <CardBody>
+                        <h4 className='title'>Argument Types</h4>
+                        <p style={{color:"#6c757d"}}>
+                            Write the space-separated value types.
+                        </p>
+                        <Row>
+                            <Col md= "8">
+                                <textarea
+                                    type="text"
+                                    value={argumentTypes}
+                                    onChange={this.handleInputChange}
+                                    placeholder="Argument Types (input example : bool address)"
+                                    name="argumentTypes"
+                                    style={{height:"160px", backgroundColor: "#adb5bd", color: "black"}}
+                                    className="form-control"
+                                />
+                            </Col>
+                        </Row>
+                        <h4 className='title'>Argument Values</h4>
+                        <p style={{color:"#6c757d"}}>
+                            Write the space-separated values to match the number of types indicated above, using square brackets [] to wrap arrays.<br></br>
+                        </p>
+                        <Row>
+                            <Col md= "8">
+                                <textarea
+                                    type="text"
+                                    value={argumentValues}
+                                    onChange={this.handleInputChange}
+                                    placeholder="Argument Values (input example : true 0x77656c636f6d6520746f20657468657265756d2e)"
+                                    name="argumentValues"
+                                    style={{height:"160px", backgroundColor: "#adb5bd", color: "black"}}
+                                    className="form-control"
+                                />
+
+                            </Col>
+                        </Row>
+                        <Row>
+                            <Col md="4">
+                                <Button style={{marginTop: "1.75rem"}} onClick={(e) => this.encodeABI(e)}>
```

Comment:
I think the margin setting should be the same as other pages, so it would be better to delete the additional marginTop setting here and ABIDecoder code.